.github/workflows/terraform.yml

```diff
@@ -0,0 +1,75 @@
+name: 'Terraform Pull Request Workflow'
+
+on:
+  pull_request:
+    paths:
+    - 'data/terraform/**'
+
+jobs:
+  terraform-check:
+    name: 'Terraform Actions'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout'
+        uses: actions/checkout@master
+
+      - name: 'Terraform Format 1'
+        uses: hashicorp/terraform-github-actions@master
+        with:
+          tf_actions_version: 0.12.13
```

Comment:
Unfortunately no :-(
(as far as I know)

Reply:
I think we can use environment variable, as in this example:
```
steps:
  - name: Hello world
    run: echo Hello world $FIRST_NAME $middle_name $Last_Name!
    env:
      FIRST_NAME: Mona
      middle_name: The
      Last_Name: Octocat
```

It would be easier to define the terraform version in one variable that can be changed in one place, then in multiple places in the code.
The verified version is pinned here: https://github.com/trawler/cna-installer/blob/e57f68e90a1f46cdbc783a9eba0015df03ae9ec3/Gopkg.toml#L30

Reply:
Ok... I changed it...
Let's see how it is working after merge...